File: 프로그래머스/1/12947. 하샤드 수/하샤드 수.java
class Solution {
    public boolean solution(int x) {
        boolean answer = true;
        String x1 = String.valueOf(x);
        int sum=0;
        for(int i=0; i<x1.length(); i++){
            sum+=Character.getNumericValue(x1.charAt(i));
        }
        if(x%sum!=0) answer=false;
        return answer;
    }
}

File: 프로그래머스/1/12947. 하샤드 수/하샤드 수.js
function solution(x) {
    var answer = true;
    var x1 = String(x);
    var sum=0;
    for(let i=0; i<x1.length; i++){
        sum+=Number(x1.charAt(i));
    }
    if(x%sum!=0) answer=false;
    return answer;
}